tests/test_add_ctg.py

```diff
@@ -0,0 +1,67 @@
+import config
+
+#Testing the 'Categories' page
+
+CTG_PAGE = "Categories"
+ADMIN_EMAIL = "admin@gmail.com"
+ADMIN_PASS = "1qaz1qaz"
+name_of_ctg = "fun"
+
+
+def test_ctg_add(app):
+    """
+        Verify that admin have ability add new category
+    """
+    expected_result = not None
+    app.landing.go_to_site()
+    app.landing.sign_up_btn.click_btn_by_css()
+    app.landing.find_event_btn.click_btn_by_css()
+    app.navigation.go_to_page(CTG_PAGE)
+    app.categories.add_ctg_btn.click_btn_by_css()
+    app.categories.ctg_name_inp.send_data(name_of_ctg)
+    app.categories.submit_btn.click_btn_by_css()
+    assert expected_result == app.categories.row_index.find_element_in_row(name_of_ctg), \
```

Comment:
actually there can be just
```suggestion
    assert app.categories.row_index.find_element_in_row(name_of_ctg) is not None, \
```
No need to create constants for None, True, False, empty lists, etc.

Reply:
done)